packages/modular-scripts/src/__tests__/lint.test.ts

```diff
@@ -97,3 +101,81 @@ describe('Modular lint', () => {
     });
   });
 });
+
+describe('lint command can successfully do selective tests based on selected packages', () => {
+  const fixturesFolder = path.join(
+    __dirname,
+    Array.from({ length: 4 }).reduce<string>(
+      (acc) => `${acc}..${path.sep}`,
+      '',
+    ),
```

Comment:
This will always resolve to `'../../../../'`. Why not simply include the raw string?

Reply:
Because `path.sep` is different in Linux | Windows | MacOS